scipy/stats/tests/test_axis_nan_policy.py

```diff
@@ -26,7 +26,8 @@
     (stats.wilcoxon, tuple(), dict(), 1, 2, True, None),
     (stats.gmean, tuple(), dict(), 1, 1, False, lambda x: (x,)),
     (stats.hmean, tuple(), dict(), 1, 1, False, lambda x: (x,)),
-    ]
+    (stats.kurtosis, tuple(), dict(), 1, 1, False, lambda x: (x,))
```

Comment:
nitpicking here, but this should have a trailing comma. That way it won't show up on VCS as a change when additional stats functions are added.